docsrc/admin/overview.md

```diff
@@ -1,22 +1,21 @@
 (overview)=
-
 # Overview
 
 This section gives an overview of the components of the Data Library system.
 
 (infrastructure)=
-
 ## Infrastructure
 
-The Data Library ansible playbook currently targets CentOS7 and CentOS Stream 9.
-IRI does not test or support the software on other
-platforms. CentOS7 is at its End of Life, and is no longer recommended to
-install.
+The Data Library ansible playbook currently targets [CentOS Stream 9](https://www.centos.org/stream9/). IRI does not test or support 
+the software on other platforms.
+
+_CentOS7 is beyond its end of life and is no longer recommended to install, but we continue to support the 
+installations where it is already installed. Any new installations must be made with CentOS Stream 9_
```

Comment:
We don't commit to supporting anyone with whom we don't have a funded project. We need CentOS 7 --> 9 upgrade instructions, and a link to them here.